js-core-1/week-3/lesson.md

```diff
@@ -1,13 +1,14 @@
-![Draft lesson](https://img.shields.io/badge/status-draft-darkred.svg)
+![Lesson in review](https://img.shields.io/badge/status-review-orange.svg)
 
 # JavaScript Core I - 3
 
-* [Array Find](#array-find)
-* [Array Some](#array-some)
-* [Array Every](#array-every)
-* [Array Filter](#array-every)
-* [Array Map](#array-map)
-* [Array ForEach](#array-foreach)
+* [Array properties](#array-properties)
+* [Manipulating arrays](#manipulating-arrays)
+* [Array methods](#array-methods)
+* [Array map](#array-map)
+* [Array forEach](#array-foreach)
+* [Array filter](#array-find)
+* [Array find](#array-find)
```

Comment:
Nice order!

Should have a small piece on higher order functions / functions as values / whatever FP name you want as an intro to explain - and then use the array props as exercises to solidify that knowledge?